packages/web/lib/browser.js

```diff
@@ -37,17 +68,15 @@ const getBrowserScope = (parent) => ({
         key,
         functions: {
           page:
-            "Defines a page definition which matches on the regex passed in as the first argument, and which executes the rule set passed in as the second argument on every matching page",
-          visit: "Open a new tab/page with the given url",
+            "Instantiates a page definition which matches on the regex passed in as the first argument, and which executes the rule set passed in as the second argument on every matching page",
+          visit:
+            "Open a new tab/page with the given url and checks for matches on that URL. If there are matches the correspond ruleSets will be run. If there is no match, a new page will be opened in the browser scope",
```

Comment:
I find this to be a little confusing...
1. as a user, what is a "browser scope". I don't think we expose the idea of a "scope" the the user anywhere, and I think that's just an implementation detail
2. "If there is no match, a new page will be opened"... does that mean, if there was a match, the page won't be opened.

iiuc, for a match, the page gets opened and the the matches are run. If there are no matches, then the `visit` call is ignored, unless you're in a repl context

is that right?

Reply:
If there is no match, the new tab is still opened although no action will be taken on the page. I updated the text to reflect that.